### PR TITLE
Update type declaration and copy in dist folder

### PR DESCRIPTION
**Pull request by riderx**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,12 +3,14 @@
...
   "description": "Random data generator with localization",
   "main": "dist/fakerator.js",
+  "types": "dist/fakerator.d.ts",
   "directories": {
     "test": "test"
   },
   "scripts": {
     "prebuild": "npm run test",
-    "build": "webpack --config config/webpack.build.config.js",
+    "copytype": "cp fakerator.d.ts dist/fakerator.d.ts",
```
Comment (icebob) [inline package.json:12]: Would be better an OS-dependent solution to copy file.

**Review by icebob — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (riderx) [reply package.json:12]: Do you have a solution in mind?
for me cp is well supported:

[All UNIX systems. Windows Server 2012. Windows 8.1. Windows Server 2012 R2. Windows 10. Windows Server 2016. Windows Server 2019.](https://www.mkssoftware.com/docs/man1/cp.1.asp)

Comment (icebob) [reply package.json:12]: Try to use it: https://github.com/calvinmetcalf/copyfiles

Comment (riderx) [reply package.json]: @icebob done !

**Pull request merged**